### ISSUE TITLE
scanvideo.c claims fixed resources

### ISSUE THREAD

**Issue by Memotech-Bill** [raspberrypi/pico-extras#60]
scanvideo.c makes claims for fixed hardware resources, specifically PIO0, low numbered DMA channels and low numbered state machines. However these may have already been claimed for other services.

Ideally the code should be modified to use the claim_unused routines.

If that is not possible, at least used fixed high number DMA channels and state machines, which would be the last resources to be claimed by the claim_unused routines.

**Comment by kilograham** [comment]
yeah, dynamic claiming would be a bit of a change, but we should at least allow the user to override all the `#define`s
